db/types.go

```diff
@@ -206,7 +204,8 @@ type OrgMember struct {
 	// OrgAddress can be used for future sharding
 	OrgAddress     common.Address `json:"orgAddress" bson:"orgAddress"`
 	Email          string         `json:"email" bson:"email"`
-	Phone          *Phone         `json:"phone" bson:"phone"`
+	Phone          HashedPhone    `json:"phone" bson:"phone"`
```

Comment:
It seems a bit confusing in other parts of the code reading _Phone_ and meaning the _HashedPhone_, since in the API the text input is also named _Phone_

What do you think?